trackData/communication.json

```
@@ -52,20 +52,21 @@
     "examples": [
       "Lead off-site workshop on interviewing",
       "Wrote WMDE's growth framework and rationale",
-      "Aligned the entire organization around claps"
+      "Aligned the entire organization around a difficult goal or steps needed to be taken"
     ]
   }, {
     "points": 20,
     "summary": "Influences outcomes at the highest level, moves beyond mere broadcasting, and sets best practices for others",
     "signals": [
       "Defines processes for clear communication for the entire team",
       "Shares the right amount of information with the right people, at the right time",
-      "Develops and delivers plans to execs, the board, and outside investors"
+      "Develops and delivers plans to managers, the board, and outside partners"
     ],
     "examples": [
       "Organized half year check-in company offsite",
       "Created the communication plan for a large organizational change",
-      "Presented to the board about key company metrics and projects"
+      "Presented to the board about key company metrics and projects",
```

do we have a board at wmde?

```suggestion
      "Presented about key metrics and projects",
```